after/plugin/dracula.vim

```diff
@@ -33,4 +33,16 @@ endif
 
 "}}}
 
+" ALE: {{{
+if exists('g:ale_enabled')
+  hi! link ALEError DraculaErrorLine
+  hi! link ALEWarning DraculaWarningLine
+  hi! link ALEInfo DraculaInfoLine
+
+  hi! link ALEErrorSign DraculaError
```

Comment:
```suggestion
```

Reply:
This is already linked to `Error` which is better than setting explicitly in the event that the person changes themes.